### ISSUE TITLE
allow backwards range in level 8

### ISSUE THREAD

**Issue by Felienne** [Felienne/hedy#606]
That would make the fishes song easier, we can later add this as a level (when we do step, too)

**Comment by thiagoarrais** [comment]
Can we have an example? Would we use the Python stepping syntax?

**Comment by Felienne** [comment]
What do you think about:

`for i in range 10 to 1`

I have always found it weird that in Python I have to do range(10,1,-1) while it could know that 10 is more than 1 so I am going backwards.

Might be a bit trickier to build but easy for kids!

**Comment by thiagoarrais** [comment]
That looks good! I believe the same should hold for level 11. `range 10 to 1` should turn into `range(10, 1)`.

**Comment by Felienne** [comment]
> That looks good! I believe the same should hold for level 11. `range 10 to 1` should turn into `range(10, 1)`.

Yes! And then at one point we need to explain to kids (maybe in a fresh level or somewhere in text in a high level) that regular Python does not do this.
